src/main/java/ro/secur/auth/configuration/SecurityConfiguration.java

```diff
@@ -0,0 +1,74 @@
+package ro.secur.auth.configuration;
+
+
+import static ro.secur.auth.common.Commons.ADMIN;
+import static ro.secur.auth.util.Api.*;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.dao.DaoAuthenticationProvider;
+import org.springframework.security.config.BeanIds;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import ro.secur.auth.security.filter.AuthenticationFilter;
+import ro.secur.auth.security.filter.TokenVerifierFilter;
+import ro.secur.auth.service.UserService;
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfiguration extends WebSecurityConfigurerAdapter {
+
+    private final UserService userService;
+
+    private final JwtConfiguration jwtConfiguration;
+
+    public SecurityConfiguration(UserService userService, JwtConfiguration jwtConfiguration) {
+        this.userService = userService;
+        this.jwtConfiguration = jwtConfiguration;
+    }
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http
+                .csrf().disable()
+                .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS)
+                .and()
+                .addFilter(new AuthenticationFilter(authenticationManager(), jwtConfiguration))
+                .addFilterAfter(new TokenVerifierFilter(jwtConfiguration), AuthenticationFilter.class)
+                .authorizeRequests()
+                .antMatchers(USERS_URL).hasRole(ADMIN)
```

Comment:
please use the enums from Role class and then convert it to string. You can create an utilitary method that prepend to the enum name the string "ROLE_"

Reply:
public static String getRole(Role role) {
        return "ROLE_" + role.toString();
    }